### PR TITLE
Let user be able to disable browserstack video recording

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by ishitatsuyuki** [comment]
@JakeChampion Please sign the CLA.

@dignifiedquire Please take a look!

**Comment by JakeChampion** [comment]
I signed it!

**Comment by ishitatsuyuki** [comment]
@dignifiedquire friendly ping, please review!

File: index.js [hunk 1]
```diff
@@ -191,7 +191,8 @@ var BrowserStackBrowser = function (
         process.env.CI_BUILD_TAG ||
         process.env.TRAVIS_BUILD_NUMBER ||
         process.env.CIRCLE_BUILD_NUM ||
-        process.env.DRONE_BUILD_NUMBER || null
+        process.env.DRONE_BUILD_NUMBER || null,
+      video: bsConfig.video !== undefined ? bsConfig.video : true
```
Comment (dignifiedquire) [inline index.js]: could you add a note about this to the readme please?

Comment (JakeChampion) [inline index.js]: Done.

**Comment by dignifiedquire** [comment]
Thank you

**Comment by oliviertassinari** [comment]
I'm curious, what's the point of disabling the video? Can we expect perf improvement?

**Pull request merged**